server/utils/referrals/caseListUtils.ts

```diff
@@ -103,96 +103,112 @@ export default class CaseListUtils {
+          case 'Conditional release date':
+            row.push({
+              attributes: { 'data-sort-value': summary.sentence?.conditionalReleaseDate },
+              text: summary.sentence?.conditionalReleaseDate
+                ? DateUtils.govukFormattedFullDateString(summary.sentence.conditionalReleaseDate)
+                : 'N/A',
+            })
+            break
+          case 'Date referred':
+            row.push({
+              attributes: { 'data-sort-value': summary.submittedOn },
+              text: summary.submittedOn ? DateUtils.govukFormattedFullDateString(summary.submittedOn) : 'N/A',
+            })
+            break
+          case 'Earliest release date':
+            row.push({
+              attributes: {
+                'data-sort-value': summary.earliestReleaseDate,
+              },
+              text: summary.earliestReleaseDate
+                ? DateUtils.govukFormattedFullDateString(summary.earliestReleaseDate)
+                : 'N/A',
+            })
+            break
+          case 'Name / Prison number':
+            row.push({
+              attributes: {
+                'data-sort-value': CaseListUtils.formattedPrisonerName(summary.prisonerName),
+              },
+              html: CaseListUtils.nameAndPrisonNumberHtml(summary),
+            })
+            break
+          case 'Parole eligibility date':
+            row.push({
+              attributes: { 'data-sort-value': summary.sentence?.paroleEligibilityDate },
+              text: summary.sentence?.paroleEligibilityDate
+                ? DateUtils.govukFormattedFullDateString(summary.sentence.paroleEligibilityDate)
+                : 'N/A',
+            })
+            break
+          case 'Programme location':
+            row.push({
+              attributes: { 'data-sort-value': summary.prisonName },
+              text: summary.prisonName || 'N/A',
+            })
+            break
+          case 'Programme name':
+            row.push({
+              text: summary.courseName,
+            })
+            break
+          case 'Programme strand':
+            row.push({
+              text: summary.audiences.map(audience => audience).join(', '),
+            })
+            break
+          case 'Progress':
+            row.push({
+              text: `${summary.tasksCompleted || 0} out of 4 tasks complete`,
+            })
+            break
+          case 'Referral status':
+            row.push({
+              attributes: {
+                'data-sort-value': summary.status,
+              },
+              html: this.statusTagHtml(summary.status),
+            })
+            break
+          case 'Release date type':
+            row.push({
+              attributes: {
+                'data-sort-value': summary.sentence?.nonDtoReleaseDateType,
+              },
+              text: summary.sentence?.nonDtoReleaseDateType
+                ? {
+                    ARD: 'Automatic Release Date',
```

Comment:
Might be worth extracting this out into some constants-type file as we may want to reference this elsewhere in future. It could live in the `utils` directory like [this one](https://github.com/ministryofjustice/hmpps-approved-premises-ui/blob/main/server/utils/allAPTypes.ts)

Reply:
Yeah - perhaps best to wait the need emerged though, unless we already have one planned?

Reply:
sure, I'm not sure if there's anywhere else we're displaying these values